lib/util.c

```diff
@@ -77,6 +77,15 @@ duo_common_ini_handler(struct duo_config *cfg, const char *section,
                 free(buf);
                 return (0);
             }
+            while (p[strlen(p) - 1] == '\\') {
```

Comment:
I'm not certain how strtok would respond to some calls being inside of a function and some outside. Even if it does work, it sounds like it would make the code more confusing. I added in a comment so it is a little more obvious what the while loop is doing.